docs/en/Debugging-Guide.md

```diff
@@ -140,14 +140,19 @@ Please read [Configuration-Settings](Configuration-Settings.html) for how to cus
 
 A: This error indicates insufficient space left on Alluxio workers to complete your write request.
 
-- Before version 1.6.0, if you are copying a file to Alluxio using `copyFromLocal`, by default this shell command applies `LocalFirstPolicy`
-and stores data on the local worker (see [location policy](File-System-API.html#location-policy)).
+- If you are copying a file to Alluxio using `copyFromLocal` with `LocalFirstPolicy`,
```

Comment:
I suggest we talk about the default policy of the latest version at beginning, and move this description on potential error of `LocalFirstPolicy` to the paragraph of `before version 1.6.0`.